pages/1.11/installing/production/system-requirements/ports/index.md

```diff
@@ -1,36 +1,42 @@
 ---
 layout: layout.pug
 title: DC/OS Ports
+navigationTitle: Ports
 menuWeight: 5
 excerpt: Making sure ports are available for installation
-oss: true
-enterprise: false
 ---
 
 [DC/OS components](/docs/1.11/overview/architecture/components/) listen on multiple ports on each node. These ports must be available for installation to succeed.
 
 DC/OS allocates additional ports to services running on top of DC/OS. These ports are required to be available when services are installed.
 
-**Important:** These ports must not be used in a firewall configuration between nodes or cluster zones.
+**Important:** 
+- These ports must not be used in a firewall configuration between nodes or cluster zones.
+- For DC/OS to install and function as intended, these ports are open and accessible upon initial installation. 
+- Therefore, network-specific security measures - from outside of the cluster as well as between internal cluster nodes and zones - for each of these ports should be evaluated and, if necessary, put in place by the network administrator before installing and implementing DC/OS. Moreover, DC/OS security modes ("disabled", "permissive", and "strict") do not affect access to these ports.
 
 ## All nodes
 
 ### TCP
 
 | Port | DC/OS Component | systemd Unit | Source | Destination |
 |---|---|---|---|---|
-| 53 | DC/OS Net | `dcos-net.service` | agent/master | agent/master |
+| 53    | DC/OS Net | `dcos-net.service` | agent/master | agent/master | 
 | 61003 | REX-Ray | `dcos-rexray.service` | agent/master (may change due to specific REX-Ray configuration)| agent/master (may change due to specific REX-Ray configuration) |
+| 61053 | Mesos DNS | `dcos-mesos-dns.service` | agent/master | master | 
 | 61091 | dcos-metrics | `dcos-metrics-agent.service/dcos-metrics-master.service` | agent/master | agent/extra |
 | 61420 | DC/OS Net | `dcos-net.service` | agent/master | agent/master |
 | 62080 | DC/OS Net | `dcos-net.service` | agent/master | agent/master |
 | 62501 | DC/OS Net | `dcos-net.service` | agent/master | agent/master |
 
+
 ### UDP
 
 | Port | DC/OS Component | systemd Unit | Source | Destination |
 |---|---|---|---|---|
-| 53 | DC/OS Net | `dcos-net.service` | agent/master | agent/master |
+| 53    | DC/OS Net | `dcos-net.service` | agent/master | agent/master [oss type="inline" size="small" /] | 
```

Comment:
@anithamuthu79 could you please remove `oss` and `enterprise` labels here?  dc/os uses 53, 61053 and 64000 in OS and EE.

Reply:
@anithamuthu79 could you please remove udp port 61053 from here? mesos dns works only on master nodes.

Reply:
@anithamuthu79 ^

Reply:
@urbanserj done.